### DISCUSSION TITLE
define "section"

### DISCUSSION THREAD

**Discussion by ruslanbes** [gethyas/hyas#294] [Q&A] [answered]
Hi,

I'm new to Hugo so sorry if the question is obvious.

I'm confused about this code in the https://github.com/h-enk/hyas/blob/master/layouts/index.html

```
{{ define "section" }}
<section class="section container-fluid">
  {{- .Content -}}
</section>
{{ end }}

```

This code affects the homepage as I understand, but If I build the site I don't see this block at all. 

I wonder how is it used and why the content is included as `{{- .Content -}}` and not `{{ .Content }}` ?

**Comment by h-enk** [comment, marked as answer]
Yes, I noticed that too recently. That section does not do anything, because there's no `block` defined for it in https://github.com/h-enk/hyas/blob/master/layouts/_default/baseof.html So, it can be /should be removed.

Trimming [whitespace ](https://gohugo.io/templates/introduction/#whitespace) on `.Content` is not necessary — I now would go for `{{ .Content }}`

**Reply by ruslanbes** [reply]
Oh I got it, I've also read about the `baseof` [in the docs](https://gohugo.io/templates/base/#define-the-base-template) and now I understand how the full HTML is being built. That was the missing piece for me.

Thanks for the clarification!

**Reply by h-enk** [reply]
236278aceea3e3021abd1ab09300081f283e4027
